src/lib/containers/RssFeedCards.tsx

```diff
@@ -0,0 +1,172 @@
+                  <p>
+                    Subscribe to receive the {this.props.mailChimpListName} by e-mail,
+                    which provides information and updates related to the
+                    Portal. You can opt out at any time by using the unsubscribe
+                    link within the e-mail. We will not share your information
+                    with any third parties or use it for any other purposes.
+                  </p>
+                  <div className="SRC-marginBottomTop">
+                    <MailchimpSubscribe url={this.props.mailChimpUrl} />
+                  </div>
+                </div>
+            )}
+          </div>
+        )}
+        <div className="RssFeed">
+          <div className="RssFeedItems">
+          {this.state.rssFeed.items &&
+            this.state.rssFeed.items.map((item: any, index: any) => {
+              // The other is to hide the large number of items in a particular feed (usually a max of 10 are returned).  See state.isShowingMoreItems
+              let parsedHtml = parser.parseFromString(
+                item['content:encoded'],
+                'text/html',
+              )
+              let bodyElement = parsedHtml.querySelector('body')
+              let moreElement = this.props.showMoreElements && parsedHtml.querySelector('[id^="more-"]')
+              if (moreElement && bodyElement) {
+                let foundMoreElement = false
+                const children = bodyElement.children
+                
+                for (let i = 0; i < children.length; i++) {
+                  let child = children[i]
+                  if (foundMoreElement) {
+                    child.innerHTML=''
+                  }
+                  if (child === moreElement) {
+                    foundMoreElement = true
+                  }
+                }
+              }
+              let isItemVisible: boolean =
+                index < this.props.defaultItemsToShow ||
+                this.state.isShowingMoreItems
+
+              return (
+                <div
+                  key={item.guid}
+                  className={`RssFeedItem ${isItemVisible ? '' : 'hidden'}`}
+                >
+                  <div>
+                    <div className="RssFeedItemCategories">
+                    {item['categories'].map((categoryName: any, index: any) => {
+                      return <div key={`${item.guid}_${categoryName}`} className="RssFeedItemCategory">{categoryName}</div>
+                    })}
+                    </div>
+                    <p className="RssFeedItemDate">
+                      {moment(item['isoDate']).format('MMMM YYYY')}
+                    </p>
+                    <p className="RssFeedItemTitle">{item['title']}</p>
+                    <div className="RssFeedItemDescription"
+                    >{parsedHtml.documentElement.innerText}</div>
+                    <a className="RssFeedItemLink" href={item['link']} target="_blank">Continue reading</a>
```

Comment:
in a follow up pr if we could set rel="noopener/noreferre", codescanning is giving an error.